extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/InnertubeClientRequestInfo.java

```diff
@@ -106,17 +109,27 @@ public static InnertubeClientRequestInfo ofWebClient() {
     public static InnertubeClientRequestInfo ofWebEmbeddedPlayerClient() {
         return new InnertubeClientRequestInfo(
                 new InnertubeClientRequestInfo.ClientInfo(WEB_EMBEDDED_CLIENT_NAME,
-                        WEB_REMIX_HARDCODED_CLIENT_VERSION, EMBED_CLIENT_SCREEN,
-                        WEB_EMBEDDED_CLIENT_ID, null),
+                        WEB_EMBEDDED_CLIENT_VERSION, WEB_EMBEDDED_CLIENT_ID, EMBED_CLIENT_SCREEN,
```

Comment:
This is the one you fixed right? I tested an age restricted video and it was properly reported as such so we should be fine I guess